cfpb/capital-framework-sandbox
# Pagination update

Part of cleaning up our recommended pagination markup and styling, which began with cfpb/capital-framework#463

## Additions

- Examples for each potential button state
- Examples for "Previous/Next" labeling

## Removals

- Obsolete "Original Documentation Markup" example

## Changes

- Updates all instances of webfont mixin calls to use the `u-` prefix.

## Testing

- Pull down the latest `canary` branch of capital-framework and run `npm run cf-link` on it
- Pull this branch locally and run `npm run cf-link`
- Run `npm start`
- Check out the examples at http://localhost:3000/components/cf-pagination/

[Preview this PR without the whitespace changes](?w=0)

## Screenshots

![screen shot 2017-03-30 at 16 52 20](https://cloud.githubusercontent.com/assets/1044670/24561985/10dea92e-1617-11e7-8ed0-317350c76c04.png)


## Checklist
